### DISCUSSION TITLE
Single Codecept Project spanning multiple Test Suites

### DISCUSSION THREAD

**Discussion by AndyMcK1** [PeterNgTr/codeceptjs-testrail#31] [Q&A]
Hi there,
with out current set up we have 1 Codecept Project that hold all of our Test cases for multiple test suites within Test Rails.
Using Pipelines we can select which Test suite we want to execute by using Tags within the individual Test scripts (eg @diary, @topusecases, etc). All of our Test scripts do already contain the relevant Test Rails test ID, our various suite all reside within the same Test rails Project.
You mention having to specify a Suite ID within the codecept config file, is there a way of passing in multiple SuiteIDs that would work for our set up or is this currently not possible

Thanks in Advance
Andy

**Comment by Hristovski** [comment]
Is there any solution to this? Have you found a workaround?
I have similar problem and I tried few options, but nothing works so far...
I have a Test Rail project with different suites for every functionality and a separate test suite file.

![image](https://user-images.githubusercontent.com/4580306/177350127-12a6dccc-004b-48ff-b844-f31bdedc4ef9.png)
7in .env file I have: **TEST_RAIL_SUITE_ID=5205**
in _codecept.conf.js_ file we have: **plugins: {testrail: { suiteId: process.env.TEST_RAIL_SUITE_ID || '1', }, }, }** etc.
so, in my tests I tried: **Feature('Smoke Tests').config({"plugins": {"testrail": {"suiteId": "5" }}})** but it doesn't work.
I tried with Before Suite
I still can't get to either override the .env file or _codecept.conf.js_ file...

Any solutions are welcome
Thanks

**Comment by jlabarrete** [comment]
Is there a way we can dynamically change or override  the plugin details in codeceptjs.config file through command line? 
In a way, I would like to change the "runName" or the "plan.name" ? 

By using a command line, 

npm run codeceptjs tests/ -runName "regression Test Sprint 1" 

I can override the value of "runName" in the plugin. 


Is this possible? 

I appreciate your reply to this.
